typings/index.d.ts

```diff
@@ -2924,6 +2925,13 @@ declare module 'discord.js' {
 
   type TargetUser = number;
 
+  interface TypingData {
+    user: User;
```

Comment:
```suggestion
    user: User | PartialUser;
```
if partials are enabled, this could be a partial user